mucommander-viewer-text/src/main/java/com/mucommander/viewer/text/TextEditorImpl.java

```diff
@@ -311,8 +312,42 @@ void selectAll() {
         textArea.selectAll();
     }
 
-    void requestFocus() {
-        textArea.requestFocus();
+    List<String> getSyntaxStyles() {
+        List<String> syntaxList = new ArrayList<>();
+        // TODO remove reflection when/if https://github.com/bobbylight/RSyntaxTextArea/issues/479 implemented
+        Field[] fields = SyntaxConstants.class.getFields();
+        for (Field field : fields) {
+            String name = field.getName();
+            if (name.startsWith("SYNTAX_STYLE_")) {
+                String[] parts = name.substring("SYNTAX_STYLE_".length()).split("_");
+                StringBuilder prettyName = new StringBuilder(parts[0]);
+                if (parts.length > 1) {
+                    for (int i = 1; i < parts.length; i++) {
+                        prettyName.append(" ");
+                        prettyName.append(parts[i].toLowerCase());
+                    }
```

Comment:
I didn't try that but seems we can replace 6 line with a single one like:
```java
Stream.of(parts).map(String::toLowerCase).collect(Collectors.joining(" "))
```
no?